recipes/apbs/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "apbs" %}
+{% set version = "1.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+    git_rev: apbs-{{ version }}
```

Comment:
Can you build from a tarball instead? See: http://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos

Reply:
They are using sub.modules intensively and the provided release tarball does not contain those sources.

Reply:
That makes, maybe make an issue about it upstream if there isn't already one?